### DISCUSSION TITLE
Hot reload

### DISCUSSION THREAD

**Discussion by friuns2** [tegal1337/YOMEN#26] [General]
Is there way to hot reload main.js without restarting chrome browser?

**Comment by fdciabdul** [comment]
> Is there way to hot reload main.js without restarting chrome browser?

i made this version , yes it can be hot reload 

https://www.youtube.com/watch?v=C7AOauxguSc&t=2s

**Reply by friuns2** [reply]
how? I run  "node main.js"
then I modify main.js but it doesn't reload it
